Modules/StudyProgramme/test/model/Progress/ilStudyProgrammeProgressTest.php

```diff
@@ -51,26 +51,27 @@ public function test_amount_of_points()
     public function test_amount_of_points_invalid()
     {
         $this->expectException(\ilException::class);
-        $spp = (new ilStudyProgrammeProgress(123))->setAmountOfPoints(-321);
+        $spp = (new ilStudyProgrammeProgress(123))->withAmountOfPoints(-321);
     }
 
     /**
      * @depends test_init_and_id
      */
     public function test_current_amount_of_points()
     {
-        $spp = (new ilStudyProgrammeProgress(123))->setCurrentAmountOfPoints(321);
+        $spp = (new ilStudyProgrammeProgress(123))->withCurrentAmountOfPoints(321);
         $this->assertEquals($spp->getCurrentAmountOfPoints(), 321);
     }
 
     public function status()
     {
         return [
-            [ilStudyProgrammeProgress::STATUS_IN_PROGRESS],
-            [ilStudyProgrammeProgress::STATUS_COMPLETED],
-            [ilStudyProgrammeProgress::STATUS_ACCREDITED],
-            [ilStudyProgrammeProgress::STATUS_NOT_RELEVANT],
-            [ilStudyProgrammeProgress::STATUS_FAILED]
+            //status, successful
+            [ilStudyProgrammeProgress::STATUS_IN_PROGRESS, false],
+            [ilStudyProgrammeProgress::STATUS_COMPLETED, true],
+            [ilStudyProgrammeProgress::STATUS_ACCREDITED, true],
+            [ilStudyProgrammeProgress::STATUS_NOT_RELEVANT, false],
```

Comment:
Wahrscheinlich verstehe ich es aktuell noch nicht, aber was ist hier "successfull"? Warum ist not_relevant ~ successfull.

Reply:
Moin. Erstmal Danke für's drüberschauen, ist viel...
COMPLETED und ACCREDITED entsprechen "successful=true".
Meine Annahme zu "not relevant" ist nicht allzu fest, ich meine aber, dass "nicht relevant" ungleich Erfolg ist. IMO muss das bei den erreichten Punkten ebenso raus wie bei den verfügbaren.